web/beacon-app/src/features/onboarding/layout/index.tsx

```diff
@@ -8,8 +8,8 @@ type Props = {
 const OnboardingFormLayout = ({ children }: Props) => {
   return (
     <>
-      <Container max={696} centered className="m-10 mt-20 px-4 xl:mt-10 xl:px-28">
-        <div className="border-[1px] border-[#72A2C0] p-10">{children}</div>
+      <Container max={696} centered className="mt-10 flex  px-4 xl:m-10 xl:mt-20 xl:px-28">
```

Comment:
Will additional changes be made to enhance the onboarding form's responsiveness in a separate story? I noticed the Log Out link still does not appear on smaller screens. Also, I noticed some other things and here's a link to a screen recording: https://www.awesomescreenshot.com/video/21122523?key=30eca620ff344d3a41adcc5aa10bf00f